### PR TITLE
Remove duplicated Ethash::init and NoProof::init calls

### LINKED ISSUE #5187: Duplicated call to Ethash::init() and NoProof::init()
These functions are called first in `aleth/main.cpp` `main()` and then called again in `MinerCLI::MinerCLI`. They register seal engines and I think they do no harm, just duplicate work. Which one should be removed?

### PR DESCRIPTION
**Pull request by tiendq**
close #5187

### CODE REVIEW

File: aleth/MinerAux.h [hunk 1]
```diff
@@ -52,11 +52,8 @@ class MinerCLI
         Benchmark
     };
 
-
     explicit MinerCLI(OperationMode _mode = OperationMode::None): mode(_mode)
     {
-        Ethash::init();
-        NoProof::init();
         BasicAuthority::init();
```
Comment (chfast) [inline aleth/MinerAux.h]: We probably should also drop `BasicAuthority::init()`.

Comment (tiendq) [reply aleth/MinerAux.h]: Ok, I'll remove it too. By the way, I just found these `init` in `WebThreeDirect::WebThreeDirect` too. `WebThreeDirect` is initialized in `main` at line 854 so I'm pretty sure those init are redundant too, but I need your confirmation.

Comment (tiendq) [reply aleth/MinerAux.h]: [The inits in `WebThreeDirect` ctor](https://github.com/ethereum/aleth/blob/master/libwebthree/WebThree.cpp#L49)

Comment (chfast) [reply aleth/MinerAux.h]: @gumb0 can you look here. Should WebTreeDirect init seal engines? Do we need to init BasicAuthority engine? It seems to be abandoned.

Comment (gumb0) [reply aleth/MinerAux.h]: Some unit tests create `WebThreeDirect` instances, but it seems testeth code calls these `init`s anyway for each test. So I think we can remove it from `WebThreeDirect` if tests don't break

Comment (gumb0) [reply aleth/MinerAux.h]: Seems that there are some unit tests for `BlockChain` that use `BasicAuthority` (`BlockChainInsertTests`)

@winsvega Do you know if `BasicAuthority` is useful for anything?

Comment (winsvega) [reply aleth/MinerAux.h]: Without this init. Windows was having build errors

Comment (winsvega) [reply aleth/MinerAux.h]: My theory is that visual studio does not build cpp file if nothing from this file is called.

Comment (tiendq) [reply aleth/MinerAux.h]: @gumb0 I have removed those inits, from `WebThreeDirect` too. No failed unit tests.

@chfast meant to drop `BasicAuthority::init()` from `MinerCLI` ctor, not the `BasicAuthority` itself. I just checked and see `BasicAuthority` is used in [BlockChainInsert.cpp](https://github.com/ethereum/aleth/blob/master/test/unittests/libethereum/BlockChainInsert.cpp) test.

**Pull request merged**